docs/developer/protocol/commander_core.md

```diff
@@ -0,0 +1,206 @@
+# Corsair Commander Core Protocol
+
+### Compatible devices
+
+| Device Name | USB ID | LED channels | Fan channels | Temperature channels | 
+|:-----------:|:------:|:------------:|:------------:|:--------------------:|
+| Commander Pro | `1B1C:0C1C` | 7 | 6 | 2 |
```

Comment:
Was this a copy paste error or is this actually compatible with the commander pro?

Reply:
I missed this; now fixed in 5534d7eda911.